src/main/java/com/amuzil/omegasource/magus/input/InputModule.java

```diff
@@ -0,0 +1,70 @@
+package com.amuzil.omegasource.magus.input;
+
+import com.amuzil.omegasource.magus.radix.Condition;
+import com.amuzil.omegasource.magus.radix.condition.minecraft.forge.EventCondition;
+import com.amuzil.omegasource.magus.skill.conditionals.InputData;
+import com.amuzil.omegasource.magus.skill.forms.Form;
+import com.amuzil.omegasource.magus.skill.modifiers.api.ModifierData;
+import com.amuzil.omegasource.magus.skill.modifiers.api.ModifierListener;
+import com.mojang.blaze3d.platform.InputConstants;
+import net.minecraft.nbt.CompoundTag;
+import net.minecraftforge.client.event.InputEvent;
+import org.apache.logging.log4j.LogManager;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public abstract class InputModule {
+    protected final Map<Condition, Form> _formInputs = new HashMap<>();
+    protected final List<ModifierListener> _modifierListeners = new ArrayList<>();
```

Comment:
These variable names don't follow our naming conventions

Reply:
I'll talk to Aidan about them, but I'll have a gander and adjust them

Reply:
Fixed